build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +53,7 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable flexible IPAM mode for Antrea. This mode allows to assign subnets to deployments and
+# stateful sets via IP Pool annotation.
+# flexibleIPAM: false
```

Comment:
the first blank space in `# flexibleIPAM: false` should be removed to `#flexibleIPAM: false` to follow the same setting as other configs. I suppose it's to allow user to remove `#` only to enable the feature.

Reply:
Good point, done